src/meta/src/hummock/compaction/tier_compaction_picker.rs

```diff
@@ -467,7 +468,15 @@ pub mod tests {
     fn test_selecting_key_range_overlap() {
         // When picking L0->L1, all L1 files overlapped with selecting_key_range should be picked.
 
-        let picker = TierCompactionPicker::default();
+        let picker = TierCompactionPicker::new(
+            0,
+            1,
+            Arc::new(CompactionConfig {
+                compaction_mode: CompactionMode::RangeMode,
+                ..Default::default()
+            }),
+            Arc::new(RangeOverlapStrategy::default()),
+        );
```

Comment:
Why are all the tests using `RangeMode` instead of `ConsistentHashMode`?

Reply:
> Why are all the tests using `RangeMode` instead of `ConsistentHashMode`?

because when they are written, they are using `RangeMode`

Reply:
So what's the use of `ConsistentHashMode`? 🤣

Reply:
We may also add tests for HashOverlapInfo.

Reply:
> So what's the use of `ConsistentHashMode`? 🤣

luckily, now we've moved to `ConsistentHashMode` in test!